### DISCUSSION TITLE
Crop or analog to cut space around image?

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [ImageMagick/ImageMagick#3944] [Help]
I need to cut white space around image. 
From this:
![1](https://user-images.githubusercontent.com/67605008/125898835-1280ced4-04fb-4131-aaa6-873aa0ad8089.png)

To like this:
![2](https://user-images.githubusercontent.com/67605008/125898840-7780fe49-1f29-46a7-8054-5503c2ef7806.png)

Any advice?

**Comment by jaka1-newbie** [comment]
mMain problem - i dont know exqct size of image-puzzle

**Comment by snibgo** [comment]
What version of IM, on what platform?

How about:
```
magick in.png -trim +repage out.png
```
See https://imagemagick.org/script/command-line-options.php#trim

**Reply by jaka1-newbie** [reply]
Yes! 
Can IM work with all images in Folder at one command?

**Reply by fmw42** [reply]
See mogrify (magick mogrify for IM 7) at https://legacy.imagemagick.org/Usage/basics/#mogrify
